utils/crypto/bls.go

```diff
@@ -0,0 +1,24 @@
+package crypto
+
+import (
+	"github.com/cornelk/hashmap"
+	"github.com/herumi/bls-eth-go-binary/bls"
+)
+
+var blsPublicKeyCache = hashmap.New[string, bls.PublicKey]()
```

Comment:
lol that's exactly what i just used, committed
even though, i'm not sure if it's even necessary because it should never go above the maximum amount of validators+operators we have, because thats what we call this func for